lib/src/fields/text_form_field.dart

```diff
@@ -90,14 +90,14 @@ class _TextJFormFieldState extends State<TextJFormField> {
             },
             style: widget.property.readOnly
                 ? const TextStyle(color: Colors.grey)
-                : WidgetBuilderInherited.of(context).uiConfig.label,
+                : Theme.of(context).textTheme.titleMedium,
```

Comment:
I think it would be better here to use `apply` or `copyWith` on the `titleMedium` theme text style and change its color for disabled, otherwise titles will have different text style when read only which may look weird.